### PR TITLE
Document how to run the Java codegen from the Daml assistant

### LINKED ISSUE #8817: Make the Java codegen examples use the Daml assistant
The Java codegen generation as described in the [codegen](https://docs.daml.com/app-dev/bindings-java/codegen.html)  section is a bit out of date with the way that it is described in the [quickstart example](https://docs.daml.com/app-dev/bindings-java/quickstart.html#integrate-with-the-ledger).  In particular, in the former we are issuing commands via `java` directly, whereas in the latter we use the `daml` assistant.

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Downloading and running the JAR manually is so 2019.

changelog_begin
[Docs] Document how to use the Java codegen using the Daml assistant
changelog_end

Fixes #8817

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/bindings-java/codegen.rst [hunk 1]
```diff
@@ -11,47 +11,34 @@ Introduction
...
-
-  java -jar <path-to-codegen-jar>
-
-Use this command to display the help text:
+The Java codegen can be run from within your Daml project using the Daml assistant. Use this command to display the help text:
 
 .. code-block:: none
 
-  java -jar codegen.jar --help
+  daml codegen java --help
```
Comment (cocreature) [inline docs/source/app-dev/bindings-java/codegen.rst:23]: maybe also worth showing how you can configure it in daml.yaml but doesn’t have to be in this PR.

**Review by cocreature — APPROVED** [review]
Thanks!

File: docs/source/app-dev/bindings-java/codegen.rst (hunk 1 above)
Comment (stefanobaghino-da) [reply docs/source/app-dev/bindings-java/codegen.rst:23]: Oh, I didn't know you could, thanks!

Comment (stefanobaghino-da) [reply docs/source/app-dev/bindings-java/codegen.rst:23]: https://github.com/digital-asset/daml/issues/10183

**Pull request merged**